proto/xudrpc.proto

```diff
@@ -410,6 +424,21 @@ message Order {
   double hold = 10 [json_name = "hold"];
 }
 
+message Trade {
+  // The order id of the maker order involved in this trade.
+  string maker_order_id = 1 [json_name = "maker_order_id"];
```

Comment:
I think it would be best to return the `Order` messages here, which means we need to fetch the orders from the database for each trade.

Reply:
I don't think I understood.

Reply:
Basically, I don't think the order id is particularly helpful to the caller of the API. They would want to see actual information about the orders involved in the trade. Since the database is relational the `trades` table only contains references to the actual orders, so for this API call I think we should actually fetch the order information. So instead of `string maker_order_id` it should be `Order maker_order`.

See the methods defined in #802, you should be able to call those to get the order data you need. Comment here or ping me on gitter if you have more questions.